lib/deimos/tracing/datadog.rb

```diff
@@ -11,15 +11,12 @@ def initialize(config)
         raise 'Tracing config must specify service_name' if config[:service_name].nil?
 
         @service = config[:service_name]
+        @tracer = ::Datadog.respond_to?(:tracer) ? ::Datadog.tracer : ::Datadog::Tracing
```

Comment:
This feels more like a method than an instance variable. It isn't actually state that needs to be stored.